user-guide/Standard_Apps/IDP/Tutorials/IDP_Tutorial_DiscoveryAndProvisioning.md

```diff
@@ -0,0 +1,160 @@
+---
+uid: IDP_Tutorial_DiscoveryAndProvisioning
+keywords: idp tutorial, idp kata
+---
+
+# Getting started with IDP and onboarding new equipment
+
+This tutorial will show you how to install IDP and onboard new equiment.
+
+Expected duration: 20 minutes.
+
+<!-- TODO: Uncomment and fill in with correct info when Kata is released.
+> [!TIP]
+> See also: [Kata #xx: IDP Introduction](https://community.dataminer.services/courses/kata-xx/)
+-->
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.0-CU0 or higher that is
```

Comment:
I think we should mention that as of DataMiner 10.4.0 or 10.4.3, the installation is easier.

Reply:
That is implied with the Important & Note part below?